### PR TITLE
Issue/add a to release

### PR DESCRIPTION
**Pull request by wouterdb**
# Description

Added -a option to inmanta release command 

# Self Check:

Strike through any lines that are not applicable (`~~line~~`) then check the box

- [ ] Attached issue to pull request
- [X] Changelog entry
- [x] Type annotations are present
- [x] Code is clear and sufficiently documented
- [x] No (preventable) type errors (check using make mypy or make mypy-diff)
- [x] Sufficient test cases (reproduces the bug/tests the requested feature)
- [x] Correct, in line with design
- [x] End user documentation is included or an issue is created for end-user documentation (add ref to issue here: )

### CODE REVIEW

File: changelogs/unreleased/release-command-dash-a.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+description: Added `-a` option to `inmanta release` command to commit all pending changes.
+issue-nr: 2508
```
Comment (arnaudsjs) [inline changelogs/unreleased/release-command-dash-a.yml:2]: This doesn't seem to be the correct ticket. So you want to reference 5082?

File: changelogs/unreleased/release-command-dash-a.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+change-type: minor
+destination-branches: [master, iso6]
+sections:
+  minor-improvement: "{{description}}"
+  upgrade-note: "The `inmanta release` command will no longer do `git -a` by default, add the `-a` option to get the old behavior "
```
Comment (arnaudsjs) [inline changelogs/unreleased/release-command-dash-a.yml:7]:
```suggestion
  upgrade-note: "The `inmanta release` command will no longer do `git commit -a` by default, add the `-a` option to get the old behavior "
```

**Review by arnaudsjs — CHANGES_REQUESTED** [review]

File: changelogs/unreleased/release-command-dash-a.yml (hunk 1 above)
Comment (wouterdb) [reply changelogs/unreleased/release-command-dash-a.yml:2]: oh! bad copy paste!

Comment (wouterdb) [reply changelogs/unreleased/release-command-dash-a.yml:2]:
```suggestion
```

**Review by arnaudsjs — APPROVED** [review]

**Pull request closed without merging**